[2022-11-04 12:15:04,806] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: xcom_dag.t1 manual__2022-11-04T12:15:02.329997+00:00 [queued]>
[2022-11-04 12:15:04,820] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: xcom_dag.t1 manual__2022-11-04T12:15:02.329997+00:00 [queued]>
[2022-11-04 12:15:04,823] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-11-04 12:15:04,824] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-11-04 12:15:04,826] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-11-04 12:15:04,848] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): t1> on 2022-11-04 12:15:02.329997+00:00
[2022-11-04 12:15:04,856] {standard_task_runner.py:52} INFO - Started process 2275 to run task
[2022-11-04 12:15:04,864] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'xcom_dag', 't1', 'manual__2022-11-04T12:15:02.329997+00:00', '--job-id', '41', '--raw', '--subdir', 'DAGS_FOLDER/xcom_dag.py', '--cfg-path', '/tmp/tmp4he3bcmf', '--error-file', '/tmp/tmpyy3na42_']
[2022-11-04 12:15:04,866] {standard_task_runner.py:80} INFO - Job 41: Subtask t1
[2022-11-04 12:15:04,958] {task_command.py:369} INFO - Running <TaskInstance: xcom_dag.t1 manual__2022-11-04T12:15:02.329997+00:00 [running]> on host 0636e12a55b7
[2022-11-04 12:15:05,043] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=xcom_dag
AIRFLOW_CTX_TASK_ID=t1
AIRFLOW_CTX_EXECUTION_DATE=2022-11-04T12:15:02.329997+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-11-04T12:15:02.329997+00:00
[2022-11-04 12:15:05,059] {python.py:173} INFO - Done. Returned value was: None
[2022-11-04 12:15:05,072] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=xcom_dag, task_id=t1, execution_date=20221104T121502, start_date=20221104T121504, end_date=20221104T121505
[2022-11-04 12:15:05,115] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-11-04 12:15:05,196] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
